bin/deb_local

```diff
@@ -67,7 +68,7 @@ def build_locally(packager, pkg_name, options)
             raise RuntimeError, "Local rebuild of pkg #{pkg_name} failed -- see #{rebuild_log}"
         end
         puts "############### build #{pkg_name} #####################"
-        debian_package_dir = packager.packaging_dir(options[:debian_package_name])
+        debian_package_dir = packager.packaging_dir_i(options[:debian_package_name])
```

Comment:
I would prefer if you cleanup the use of _i  and replacing it by 'info', e.g. for pkgi to pkginfo, where it makes sense.
In this case for example, I would remain with packaging_dir